src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -683,6 +683,7 @@
     <Compile Include="Microsoft\Data\SqlClient\TdsParserStateObjectManaged.cs" />
     <Compile Include="Microsoft\Data\SqlTypes\SqlTypeWorkarounds.netcore.cs" />
     <Compile Include="Microsoft.Data.SqlClient.TypeForwards.cs" />
+    <Compile Include="RuntimeHelpers.cs" />
```

Comment:
Is this the correct location for this file? Or where would you put that? (It's factoring out the RuntimeHelpers from TdsParserStateObject.netcore.cs)

Reply:
I'd personally align the folder structure with the namespace, at Microsoft\Data\SqlClient\RuntimeHelpers.cs or RuntimeHelpers.netcore.cs (I'm not completely sure which - I've seen both patterns.)

Reply:
`namespace System.Runtime.CompilerServicesOverride`

I think that's the best solution that I can come up with

Reply:
I've moved it to the `Microsoft.Data.SqlClient` namespace